app/admin/practices.rb

```diff
@@ -33,6 +41,7 @@
     column :enabled unless params[:scope] == "get_practice_owner_emails"
     column :created_at unless params[:scope] == "get_practice_owner_emails"
     column 'Last Updated', :updated_at unless params[:scope] == "get_practice_owner_emails"
+    column 'Date Published', :date_published unless params[:scope] == "get_practice_owner_emails"
```

Comment:
instead of 'Date Published' you can just do `column :date_published unless params[:scope] == "get_practice_owner_emails"`

Reply:
Yeah - I didn't think of that... but I know.. Active Admin would default to what I wanted in that case..  :)